internal/blockchain/ethereum/ethereum.go

```diff
@@ -131,10 +132,13 @@ func (e *Ethereum) Init(ctx context.Context, prefix config.Prefix, callbacks blo
 		GlobalSequencer: true,
 	}
 
-	if ethconnectConf.GetString(wsclient.WSConfigKeyPath) == "" {
-		ethconnectConf.Set(wsclient.WSConfigKeyPath, "/ws")
+	if ethconnectConf.GetString(wsconfig.WSConfigKeyPath) == "" {
+		ethconnectConf.Set(wsconfig.WSConfigKeyPath, "/ws")
```

Comment:
Should we do this here, or on the generated `wsConfig` object..? probably doesn't matter either way, just pointing out that we _could_ move it down.